### ISSUE TITLE
Show controls for Arcsi player on mobile lock screen

### ISSUE THREAD

**Issue by baxgas** [mmmnmnm/lahmacun_rework#103]
@gammaw – I think we had some solution for this on the stream player?

**Comment by gammaw** [comment]
> @gammaw – I think we had some solution for this on the stream player?

Yeah, we had this https://github.com/mmmnmnm/lahmacun/issues/182 lengthy issue on the topic.

**Comment by baxgas** [comment]
@gammaw that is exactly how I made it, with MediaSession
https://github.com/mmmnmnm/lahmacun_rework/commit/f7eb8f44d5a37fd6a97bef052b9a93f74c7bca68

**Comment by gammaw** [comment]
@baxgas sure we wanna close it? We had those issues on iPhone, just checked on our test server and it still doesn't seem to wokr properly, e.g., when I switch from arcsi player to the radio player the lock screen still shows the title of the show in the arcsi player and the play/paused statuses didn't seem to correctly update.
